tfx/components/trainer/fn_args_utils.py

```diff
@@ -59,14 +59,14 @@ def get_common_fn_args(input_dict: Dict[Text, List[types.Artifact]],
                        working_dir: Text = None) -> FnArgs:
   """Get common args of training and tuning."""
   train_files = [
-      io_utils.all_files_pattern(
-          artifact_utils.get_split_uri(input_dict[constants.EXAMPLES_KEY],
-                                       'train'))
+      io_utils.all_files_pattern(uri) for uri in
+      artifact_utils.get_split_uris(input_dict[constants.EXAMPLES_KEY],
```

Comment:
Overall lgtm, could you print `input_dict[constants.EXAMPLES_KEY]` here, and paste result here? I want to check the uri and id to see if it matches what examplegen generates

Reply:
Attached is that specific log:

`
0717 20:04:34.873566 135202664122112 fn_args_utils.py:63] [Artifact(artifact: id: 1
type_id: 5
uri: "/tmp/iris_resolver_e2e_testfc2pebe5/tmpa0t72p3g/testIrisPipelineResolverWithDependency/tfx/pipelines/resolver_test/CsvExampleGen/examples/1"
properties {
  key: "split_names"
  value {
    string_value: "[\"train\", \"eval\"]"
  }
}
custom_properties {
  key: "input_fingerprint"
  value {
    string_value: "split:single_split,num_files:1,total_bytes:2758,xor_checksum:1595041459,sum_checksum:1595041459"
  }
}
custom_properties {
  key: "name"
  value {
    string_value: "examples"
  }
}
custom_properties {
  key: "payload_format"
  value {
    string_value: "FORMAT_TF_EXAMPLE"
  }
}
custom_properties {
  key: "pipeline_name"
  value {
    string_value: "resolver_test"
  }
}
custom_properties {
  key: "producer_component"
  value {
    string_value: "CsvExampleGen"
  }
}
custom_properties {
  key: "span"
  value {
    string_value: "0"
  }
}
custom_properties {
  key: "state"
  value {
    string_value: "published"
  }
}
create_time_since_epoch: 1595041459439
last_update_time_since_epoch: 1595041460679
, artifact_type: id: 5
name: "Examples"
properties {
  key: "span"
  value: INT
}
properties {
  key: "split_names"
  value: STRING
}
), Artifact(artifact: id: 4
type_id: 5
uri: "/tmp/iris_resolver_e2e_testfc2pebe5/tmpa0t72p3g/testIrisPipelineResolverWithDependency/tfx/pipelines/resolver_test/CsvExampleGen/examples/4"
properties {
  key: "split_names"
  value {
    string_value: "[\"train\", \"eval\"]"
  }
}
custom_properties {
  key: "input_fingerprint"
  value {
    string_value: "split:single_split,num_files:1,total_bytes:2758,xor_checksum:1595041464,sum_checksum:1595041464"
  }
}
custom_properties {
  key: "name"
  value {
    string_value: "examples"
  }
}
custom_properties {
  key: "payload_format"
  value {
    string_value: "FORMAT_TF_EXAMPLE"
  }
}
custom_properties {
  key: "pipeline_name"
  value {
    string_value: "resolver_test"
  }
}
custom_properties {
  key: "producer_component"
  value {
    string_value: "CsvExampleGen"
  }
}
custom_properties {
  key: "span"
  value {
    string_value: "1"
  }
}
custom_properties {
  key: "state"
  value {
    string_value: "published"
  }
}
create_time_since_epoch: 1595041464576
last_update_time_since_epoch: 1595041465895
, artifact_type: id: 5
name: "Examples"
properties {
  key: "span"
  value: INT
}
properties {
  key: "split_names"
  value: STRING
}
), Artifact(artifact: id: 7
type_id: 5
uri: "/tmp/iris_resolver_e2e_testfc2pebe5/tmpa0t72p3g/testIrisPipelineResolverWithDependency/tfx/pipelines/resolver_test/CsvExampleGen/examples/7"
properties {
  key: "split_names"
  value {
    string_value: "[\"train\", \"eval\"]"
  }
}
custom_properties {
  key: "input_fingerprint"
  value {
    string_value: "split:single_split,num_files:1,total_bytes:2758,xor_checksum:1595041469,sum_checksum:1595041469"
  }
}
custom_properties {
  key: "name"
  value {
    string_value: "examples"
  }
}
custom_properties {
  key: "payload_format"
  value {
    string_value: "FORMAT_TF_EXAMPLE"
  }
}
custom_properties {
  key: "pipeline_name"
  value {
    string_value: "resolver_test"
  }
}
custom_properties {
  key: "producer_component"
  value {
    string_value: "CsvExampleGen"
  }
}
custom_properties {
  key: "span"
  value {
    string_value: "2"
  }
}
custom_properties {
  key: "state"
  value {
    string_value: "published"
  }
}
create_time_since_epoch: 1595041469775
last_update_time_since_epoch: 1595041471049
, artifact_type: id: 5
name: "Examples"
properties {
  key: "span"
  value: INT
}
properties {
  key: "split_names"
  value: STRING
}
)]`